project_completion_report/report/project_completion_report.py

```diff
@@ -2,85 +2,110 @@
 # © 2016 Elico Corp
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from openerp.osv import fields, osv
-from openerp import tools
+from openerp import models, api, fields, tools
 
 
-class ProjectCompletionReport(osv.Model):
-    """Project Completion Report"""
+class ProjectCompletionReport(models.Model):
+    """
+        Project Completion Report
+    """
     _name = "project.completion.report"
+    # Database table should not be created, use init() instead
     _auto = False
     _description = "Project Completion Report"
+    # Field used for the Name
     _rec_name = 'activity_name'
 
-    _columns = {
-        'id': fields.integer('Id', readonly=True),
-        'activity_type': fields.selection(
-            [
-                ('task', 'Task'),
-                ('issue', 'Issue'),
-            ], 'Type', readonly=True,
-            help="Type is used to separate Tasks and Issues"),
-        'hours': fields.float(
-            'Time spent', digits=(16, 2), readonly=True,
-            help="Time spent on timesheet"),
-        'user_id': fields.many2one('res.users', 'User', readonly=True),
-        'project_id': fields.many2one(
-            'project.project', 'Project', readonly=True),
-        'project_state': fields.char(

```

Comment:
@Miya609 seems like this field has been forgotten during migration to v10, please add it back

FYI it's here in v8 new API:
https://github.com/Elico-Corp/odoo-addons/blob/8.0/project_completion_report/report/project_completion_report.py#L31

This is the view:
https://github.com/Elico-Corp/odoo-addons/blob/8.0/project_completion_report/report/project_completion_report_view.xml#L15